feature/gnmi/otg_tests/telemetry_port_speed_test/telemetry_port_speed_test.go

```diff
@@ -321,9 +320,13 @@ func (tc *testCase) configureATE(t *testing.T) {
 func (tc *testCase) verifyDUT(t *testing.T, numPort int) {
 	dutPort := tc.dut.Port(t, "port1")
 	want := int(dutPort.Speed()) * numPort * 1000
-	val, _ := gnmi.Watch(t, tc.dut, gnmi.OC().Interface(tc.aggID).Aggregation().LagSpeed().State(), 60*time.Second, func(val *ygnmi.Value[uint32]) bool { return val.IsPresent() }).Await(t)
-	if got, _ := val.Val(); int(got) != want {
-		t.Errorf("Get(DUT port status): got %v, want %v", got, want)
+	val, status := gnmi.Watch(t, tc.dut, gnmi.OC().Interface(tc.aggID).Aggregation().LagSpeed().State(), 60*time.Second, func(val *ygnmi.Value[uint32]) bool { return val.IsPresent() }).Await(t)
+	if status {
+		if got, _ := val.Val(); int(got) != want {
+			t.Errorf("Get(DUT port status): got %v, want %v", got, want)
+		}
+	} else {
+		t.Errorf("Timeout waiting for the Lagspeed value")
```

Comment:
@octpetre : Thinking we directly compare in the `Await`

```
got, ok := gnmi.Watch(t, tc.dut, gnmi.OC().Interface(tc.aggID).Aggregation().LagSpeed().State(), 60*time.Second, func(val *ygnmi.Value[uint32]) bool {
    speed, ok := val.Val()
    return ok && speed == want
}).Await(t)
if !ok {
    t.Errorf("Get(DUT port status): got %v, want %v", got, want)
}
```

Reply:
Done